.travis.yml

```diff
@@ -15,30 +15,46 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
+      services: docker
+      env: DOCKER_IMAGE=ubuntu:16.04 KITURA_NIO=1
```

Comment:
These `env:` lines need squashing into a single statement otherwise Travis will use the last one declared (looks like this has been wrong for a while).  Eg:
`      env: DOCKER_IMAGE=ubuntu:16.04 KITURA_NIO=1 TESTDB_NAME=testdb_2`